packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -940,6 +943,17 @@ export class ActiveWorkflowRunner implements IWebhookManager {
 	// TODO: this should happen in a transaction
 	async remove(workflowId: string) {
 		// Remove all the webhooks of the workflow
+
+		if (this.multiMainSetup.isEnabled && this.multiMainSetup.isFollower) {
+			this.logger.debug(
+				'[Multi-main setup] Instance is follower, skipping removal of workflow from active workflows in memory and broadcasting "workflowWasDeactivated" into command channel...',
+			);
+
+			await this.multiMainSetup.broadcastWorkflowWasDeactivated(workflowId);
```

Comment:
What should the reaction by the listening clients be when receiving this message? Iirc the idea was that when some client receives this, it also deactivates that workflow. But that means this message should only every been sent once, by the leader, not by followers, or am I mistaken here?

Reply:
Solved during pairing